ui/css/itcss/components/send.scss

```diff
@@ -1,927 +0,0 @@
-  &__error-message {
-    display: none;
-  }
-
-  &--error {
-    input,
-    .send-screen-gas-input {
-      border-color: var(--color-error-default) !important;
-    }
-
-    .send-screen-input-wrapper__error-message {
-      @include design-system.H7;
-
-      display: block;
-      position: absolute;
-      bottom: 4px;
-      left: 8px;
-      color: var(--color-error-default);
-    }
-  }
-
-  .send-screen-input-wrapper__error-message {
-    @include design-system.H7;
-
-    display: block;
-    position: absolute;
-    bottom: 4px;
-    left: 8px;
-    color: var(--color-error-default);
-  }
-}
-
-.send-screen-input {
-  width: 100%;
-}
-
-.send-screen-gas-input {
-  @include design-system.Paragraph;
-
-  width: 100%;
-  height: 41px;
-  border-radius: 3px;
-  background-color: var(--color-background-default);
-  border-width: 0;
-  border-style: none;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  padding-left: 10px;
-  padding-right: 12px;
-  color: var(--color-text-alternative);
-}
-
-.send-screen-amount-labels {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-}
-
-.send-screen-gas-labels {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-}
-
-.currency-toggle {
-  &__item {
-    color: var(--color-primary-default);
-    cursor: pointer;
-
-    &--selected {
-      color: var(--color-text-default);
-      cursor: default;
-    }
-  }
-}
-
-.send-screen-gas-input-customize {
-  @include design-system.H7;
-
-  color: var(--color-primary-default);
-  cursor: pointer;
-}
-
-.gas-tooltip-close-area {
-  position: fixed;
-  top: 0;
-  left: 0;
-  z-index: 1000;
-  width: 100%;
-  height: 100%;
-}
-
-.gas-tooltip-arrow {
-  height: 25px;
-  width: 25px;
-  z-index: 1200;
-  background: var(--color-background-default);
-  position: absolute;
-  transform: rotate(45deg);
-  left: 107px;
-  top: 294px;
-  box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
-}
-
-.gas-tooltip {
-  display: flex;
-  justify-content: center;
-}
-
-.gas-tooltip-label {
-  @include design-system.Paragraph;
-
-  color: var(--color-text-alternative);
-}
-
-.gas-tooltip-header {
-  padding-bottom: 12px;
-}
-
-.gas-tooltip-input-label {
-  margin-bottom: 5px;
-}
-
-.gas-tooltip-input-label i {
-  color: var(--color-icon-muted);
-  margin-left: 6px;
-}
-
-.gas-tooltip-input-detail {
-  @include design-system.H7;
-
-  position: absolute;
-  top: 4px;
-  right: 26px;
-  color: var(--color-text-muted);
-}
-
-.gas-tooltip-input-arrows {
-  @include design-system.H6;
-
-  position: absolute;
-  top: 0;
-  right: 4px;
-  width: 17px;
-  height: 28px;
-  border: 1px solid var(--color-icon-muted);
-  border-left: 0;
-  display: flex;
-  flex-direction: column;
-  color: var(--color-text-muted);
-  padding: 1px 4px;
-  cursor: pointer;
-}
-
-.token-gas {
-  &__amount {
-    display: inline-block;
-    margin-right: 4px;
-  }
-
-  &__symbol {
-    display: inline-block;
-  }
-}
```

Comment:
Not used

![Screenshot 2025-02-07 at 12 01 16 PM](https://github.com/user-attachments/assets/02780b6d-42af-4678-96f2-52c87822d4a4)